base/task.jl

```diff
@@ -3,7 +3,7 @@
 ## basic task functions and TLS
 
 # Container for a captured exception and its backtrace. Can be serialized.
-type CapturedException
+type CapturedException <: Exception
```

Comment:
It was designed to be a container for an exception and its backtrace that can be safely serialized. Since it was not the exception itself, I guess I didn't make it a subtype of Exception.